website/docs/docs/cloud/visual-editor-interface.md

```diff
@@ -62,7 +62,7 @@ If you have any feedback on additional operators that you might need, we'd love
 
 ## Canvas
 
-The visual editor has a sleek drag-and-drop canvas interface that allows you to create or modify dbt SQL models. It's like a digital whiteboard space that allows analysts to deliver trustworthy data. Use the canvas to:
+The visual editor has a sleek drag-and-drop canvas interface that allows you to create or modify dbt SQL models. It's like a digital whiteboard space that allows anyone to view and deliver trustworthy data. Use the canvas to:
```

Comment:
It might be wise to avoid "anyone" because they still have to have a dbt license. It's not a tool anyone can access via a link share or be granted access with just an email address (like a shared Google Sheet). Also, I'm not a fan of "allows," which I put in there originally (that's on me). How about: 
```suggestion
The Visual Editor has a sleek drag-and-drop canvas interface for creating and modifying dbt SQL models. It's like a digital whiteboard space for easily viewing and delivering trustworthy data. Use the canvas to:
```